pkg/appgateserver/server.go

```diff
@@ -222,20 +223,22 @@ func (app *appGateServer) ServeHTTP(writer http.ResponseWriter, request *http.Re
 // TODO_TECHDEBT: This method should be aware of the nature of the error to use the appropriate JSONRPC
 // Code, Message and Data. Possibly by augmenting the passed in error with the adequate information.
 func (app *appGateServer) replyWithError(writer http.ResponseWriter, err error) {
-	relayResponse := &types.RelayResponse{
-		Payload: &types.RelayResponse_JsonRpcPayload{
-			JsonRpcPayload: &types.JSONRPCResponsePayload{
-				Id:      0,
-				Jsonrpc: "2.0",
-				Error: &types.JSONRPCResponseError{
-					// Using conventional error code indicating internal server error.
-					Code:    -32000,
-					Message: err.Error(),
-					Data:    nil,
-				},
-			},
+	response := map[string]interface{}{
```

Comment:
It makes sense to have the `jsonServer` respond with a general purpose json rpc error, but it doesn't make sense in the context of the `appGateServer`.

This should be just a plain error string or something pocket specific.

Reply:
I think errors should be returned in the form of the request being sent, if ur code sending the requests is expecting a json reply lets give them a json reply or else the decoding will get weird on the app end